third_party/move/tools/move-mutator/src/lib.rs

```diff
@@ -23,13 +25,29 @@ pub fn run_move_mutator(
         options, config, package_path
     );
 
-    let (_files, ast) = generate_ast(options.move_sources, config, package_path)?;
+    let (files, ast) = generate_ast(options.move_sources, config, package_path)?;
 
     let mutants = mutate::mutate(ast)?;
 
-    println!("Mutants:");
-    for mutant in mutants {
-        println!("{}", mutant);
+    let _ = std::fs::remove_dir_all("mutants_output");
```

Comment:
maybe const can be used since we use this const at more than two locations
```suggestion
    let _ = std::fs::remove_dir_all(OUTPUT_DIR);
```

Reply:
Agree! 
Changed :+1: